contracts/Gauge.sol

```diff
@@ -150,31 +149,61 @@ contract Gauge is BaseGauge, IGauge {
 
     /** @return total of the staked vault token
      */
-    function totalSupply() public view returns (uint256) {
```

Comment:
erc4626 is also an  erc20 so this method still needs to be available to be erc20 compliant, not sure if the erc4626 interface extends from correct erc20 interface.